src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/tests/stores/PreviewStore.test.js

```diff
@@ -26,7 +26,7 @@ test('Should request server on start preview', () => {
     previewStore.start();
 
     return requestPromise.then(() => {
-        expect(Requester.post).toBeCalledWith('/start?id=123-123-123&locale=en&provider=pages');
```

Comment:
looks like the `query-string` library sorted the parameters by the parameter key